.github/ISSUE_TEMPLATE/development-task.md

```diff
@@ -0,0 +1,7 @@
+---
+name: "\U0001F680 Development Task"
```

Comment:
Isn't is should be one of bug/feature-request?

Reply:
IMO, bug typed issues should use bug report template.

Reply:
What I mean is development task either solves a bug or implements a new feature.